### PR TITLE
Removes public autolathes

### PR DESCRIPTION
**Pull request by Bawhoppen**
:cl: Powercreep Balance Committee
del: Public autolathes have been removed. Please ask (or break into) cargo if you need something.
/:cl:

I think this change should be fairly apparent why it would be beneficial, but if not:

Public autolathes are part of the large amount of resource-creep which became popular with the rise of multiple maps and map rotation. Having an over-abundance of resources, tools, and gear available to the crew stifles meaningful player interaction and conflict. 

Autolathe goods being locked behind cargo's doors would require players, once again, to interact with their fellow crew on cargo duty in order to receive what they want (or to break in and evade security). We survived for years with this mechanically being the case, and I believe it would only be logical to return to it.

Cobby: Closes #41938

### CODE REVIEW

**Comment by FrostyFridge** [comment]
he actually did it the absolute madman

**Comment by necromanceranne** [comment]
Engineering can still produce their own autolathe with relative ease, so not much really changes if they just go make one in a public area like the tool storage space. The autolathe board is not competitively fought over (except by that one assistant making .357 speed loaders in maint to make Columbine look like a joke), and nobody is going to complain if someone makes a public one again. A lot of engineers already just make their own in-shop autolathe anyway. Whether it is attached to science or engineering means very little.

Additionally, this change is really only going to affect assistants who will club each other to death over a toolbox anyway. If you want more tool-related homicides committed by greyshirts, this is how you do it. Assistants already have received some harsh 'nerfs' with the introduction of economy, because goof thought having greyshirts be literal fucking hobos a fair and reasonable thing to do. How much more strapped for resources do you want the average greyshirt to be?

**Comment by lolman360** [comment]
good pr, speedstealthtestmerge

**Comment by XDTM** [comment]
What's the point of improvised weapons if you can print circular saws right away

**Comment by Bawhoppen** [comment]
@coiax Sorry Coiax... it couldn't be avoided though sadly...

**Comment by granpawalton** [comment]
The easy access to low level tools is a lot healthier for a lot of assistant players because it removed the mechanic of making a roundstart 15 second mad dash for x, y, and z before someone else took it, which just caused stagnant interactions and headaches trying to catch back up to the starting point of the gimmick you had in mind

Besides that yeah circular saws, large beakers, ammo, and etc would be better kept behind closed doors because it is cutting out a part of the game that changed between each shift and balanced things out well

**Comment by davethwave** [comment]
Ammo for sec, medical weaponry for medical.

**Comment by ExcessiveUseOfCobblestone** [comment]
Good PR

**Comment by coiax** [comment]
I am ambivalent, because I think we should keep my BEAUTIFUL CARGO SHUTTERS on Box.

**Comment by davethwave** [comment]
Before removal, it would be a good idea to see some other department specific stuff and add them to the departmental lathes.

**Comment by Nickkoranda** [comment]
The grimdark timeline for TG where everyone is deciding to remove everything while adding nothing back in.

**Comment by ExcessiveUseOfCobblestone** [comment]
Get off assistant lol

**Comment by Nickkoranda** [comment]
god forbid I play the game without validhunting and instead use tools for autism projects

**Comment by ExcessiveUseOfCobblestone** [comment]
Engineering is the job for you!

**Comment by RogueSteampunker** [comment]
I still find it hilarious how we expect both assistants AND cargo to actually want to interact with anyone in any form that will not end in bwoink after bwoink and wondering why we tried in the first place

Cargo rarely of ever wants to work or supply without a fat load of credits in front of them, and assistants will just break in anyway. All this solves is stopping people who don't greytide as a living from getting tools cause muh cargo needs more power to abuse

**Comment by coiax** [comment]
>I still find it hilarious how we expect both assistants AND cargo to actually want to interact with anyone in any form that will not end in bwoink after bwoink and wondering why we tried in the first place

You're saying that assistants asking for some tools to be printed will result in boinks?

**Comment by AlexTheSysop42** [comment]
he's saying they won't ask

**Comment by necromanceranne** [comment]
> You're saying that assistants asking for some tools to be printed will result in boinks?

He's saying that assistants will break in and assault cargo for tools. Botanists currently already do this to chemistry. There is precedent for this.

**Comment by Shadowflame909** [comment]
I think this PR could work if it was something more then just another removal.

It's obvious that the outcome of such a thing is that Cargo's going to get tided every shift. Maybe that'll be someone stealing/moving the ORM (Mostly stealing) to get access into Cargo. Or just breaking the windows.


All I'm saying is that you should give Cargo a round start infinite Pie Cannon that only works in cargo.

**Comment by necromanceranne** [comment]
> > He's saying that assistants will break in and assault cargo for tools. Botanists currently already do this to chemistry. There is precedent for this.
> 
> And, you know, the several years we went without public lathes.

What I described was not in praise, you crusty old man.

**Comment by PKPenguin321** [comment]
And what I described was, you delightful newfriend

**Comment by XDTM** [comment]
Put it in the vault surrounded by turrets

**Comment by anconfuzedrock** [comment]
Shit assistants will be shit, giving them infinite tools just means they’ll break more shit with tools. If you hate em so much, nerf em.

**Comment by AlexTheSysop42** [comment]
By your own logic, taking away their tools just means they'll break more shit without tools

And if they really need their tools to be shit, they'll just roll engi and not do anything useful

**Comment by XDTM** [comment]
>giving them infinite tools just means they’ll break more shit with tools

>By your own logic, taking away their tools just means they'll break more shit without tools

🤔

**Comment by AlexTheSysop42** [comment]
I mean they'll just find ways to break shit without tools

he said in his own comment, shit assistants will be shit

**Comment by anconfuzedrock** [comment]
Any assistant who thinks breaking in somewhere with a fire extinguisher is the best way to get tools is an assistant who thinks hacking into the hops office with tools is fine too. I’d rather they not have rcds.

**Comment by AlexTheSysop42** [comment]
They'll just go engi and get a filled rcd from the vendor, wasting a slot and having even MORE tools and access than before. _If they want to tide they'll tide as a different role._ This is why none of the "remove assistants" ideas got very far.

**Comment by anconfuzedrock** [comment]
If that’s so true, I should be seeing all engineer slots filled every time already, but I don’t. That only works for jobbans, alex.

**Comment by RogueSteampunker** [comment]
Why would you be seeing engineer slots filling right now? There's no reason (yet) to just go engineer instead of assistant since most all the tools are still available to them

**Comment by ExcessiveUseOfCobblestone** [comment]
I think you need to talk with your local administration about greytiding instead of having the code spoonfeed everyone because the evil grey boogeyman takes all the scarce items first.

**Comment by RogueSteampunker** [comment]
I mean, considering the fact that attempting to stomp greytiding is like trying to get naming policy into an agreeable stance, good fucking luck with that.

And besides, greytiding or not, there's still only two multitools and a single youtool vendor, both of which will end up taken or stolen, usually either by the greytide or the same people we seemingly trust to not just cause more murders by holding the autolathe hostage. It's not helping anything to take the box approach, when box is the one map where a cargo window unbroken or a cargo door unhacked is a fucking sign of god himself.

**Comment by anconfuzedrock** [comment]
Gloves were always really coveted, but people had and always have gone assistant despite engineer having upsides and no downsides. Cargo is fine on box if sec does its job or even if the qm uses his flash (or his infinite spray can stuns that are available to assistants on every other station). Multitools aren't necessary to live, and life is more interesting without one. one single youtool vendor is like saying that the heavy in tf2 only starts with one full magazine of minigun ammo. I'd rather have an autolathe which has contents that make sense and that gives cargo actual cargo work at least sometimes, than have an autolathe arbitrarily having every surgical tool except the saw (also LOVE how every map but box gives science the public autolathe which they can close the shutters on even though it's cargo stuff)

**Comment by improvedname** [comment]
As soon you break a window of cargo as assistant. Your declared valid will be killed skinned organ harvested, and exported for about 5000 credits. Your brain will be send the chef for a delicous brain burger. This is the conquense if you mess with cargo.

**Comment by improvedname** [comment]
Also reasons that greytide want to keep autolathes.

Improved shotguns, 1 ammo cap
Can range up from several variations of ammo as the following.

Buckshots, Taser slugs, Shotgun darts, Ion shells, laser slugs, rubbershots, beanbags, meteorshots, pulse slugs.

Even if people call the improvised shotgun worthless. Its universality proves its strong enough and potentionally can end in a 2 shot weapon which also can be sawn off with a circular saw which you can litteraly get from the same place.

RCD, i love this bueaty. You dont need insulated gloves at all to hack a autolathe safely just use multitool and you wont be shocked at all and to top that off you can make a RCD that removes any airlock/wall/grille at a instant without any shock risk attached to it either(tools printed from the same location what a fucking suprise)

Spraypaint, litteraly ghetto flash which also applies burn stacks still a easy kill to anyone without eyewear.

Circular saw. 15 brute weapon
Surgical drill 15 brute weapon (target eyes for fun!!!)

Source of igniters for stunprods(remember when we switched from wirecutters to igniters because wirecutters were to common doesnt fucking matter with a autolathe anyways)

Anyone can import a certain design for whatever reason that may be lol but only ones with basic materials((which is tottaly fine, not like anyone does it anyways unless its free golems))

Also one of the main sources to make meteorshots which even moves anchored objects, can shoot blob core off into space etc.

Tl:dr remove public autolathes

**Comment by imsxz** [comment]
more public insuls to save the poor assistant mains from asking/breaking into engineering like they already do pls. QOL.

**Comment by ExcessiveUseOfCobblestone** [comment]
If everyone dies because no autolathes we can revert lol.

**Comment by carshalash** [comment]
C-c-c-can't wait for all of the engineer break ins now. Th-th-th-th-th-thanks coders....

**Comment by anconfuzedrock** [comment]
based and redpilled

**Comment by adamkad1** [comment]
congratz, you annoyed a lot of people, now go sit in a corner

**Comment by Bawhoppen** [comment]
@adamkad1 In my opinion this was a pretty fair balance decision. There will always be atleast some amount of people that are annoyed at any given change. I think, unfortunately, if we weren't willing to let that remain as the case, we'd never be able to make any more changes again.

**Comment by ExcessiveUseOfCobblestone** [comment]
Every round since this change has literally been the purge. I can't walk around 2 corners before finding an engineer abusing his/her position as previous greytider. 

There are people sending me death threats to my home saying if I don't revert this change they're going to delete my static character, someone help!

**Comment by adamkad1** [comment]
> Every round since this change has literally been the purge. I can't walk around 2 corners before finding an engineer abusing his/her position as previous greytider.
> 
> There are people sending me death threats to my home saying if I don't revert this change they're going to delete my static character, someone help!

Hilarious
